### PR TITLE
Add new languages and New Frameworks.

### LINKED ISSUE #1734: [feature]: update all the frameworks logo on landing page
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Summary

right now only four language are mentioned on the landing page. 
image

But keploy supports more languages such as rust, csharp, etc..

### Why should this be worked on?

This would help the first time discovererd user who are coming on main website to know more about the supports that keploy provide

### Repository

website

### PR DESCRIPTION
**Pull request by Aditya-eddy**
closes [#issue](https://github.com/keploy/keploy/issues/1734)
I have updated language to c# and rust.
If we need to update any new frameworks please do list them or any new languages we need to add.
Screenshot from 2024-04-01 10-10-40

### CODE REVIEW

**Comment by Hermione2408** [comment]
Hey @Aditya-eddy, I think we should line them up in a single row you can consider slightly reducing their size to fit

**Comment by Aditya-eddy** [comment]
How does this look?
Screenshot from 2024-04-02 09-12-34

**Comment by Hermione2408** [comment]
> How does this look? Screenshot from 2024-04-02 09-12-34

This looks fine

**Comment by Aditya-eddy** [comment]
Any new frameworks to be updated? 
@Hermione2408 , @nehagup

File: components/language.tsx [hunk 1]
```diff
@@ -502,7 +502,7 @@ export default function Language() {
           </div>
 
           {/* Items */}
-          <div className="grid max-w-sm grid-cols-4 gap-2 mx-auto md:max-w-4xl md:grid-cols-4">
+          <div className="grid max-w-sm grid-cols-6 gap-7 mx-auto md:max-w-4xl md:grid-cols-6 place-content-center">
```
Comment (Hermione2408) [inline components/language.tsx]: please reduce this gap looking too much in small screen

**Review by Hermione2408 — CHANGES_REQUESTED** [review]
Rust image is not looking centered align can you please check .Also you can add your command CLI space changes in this PR.

**Comment by Aditya-eddy** [comment]
Does this changes look good?
Screenshot from 2024-04-03 09-24-25
Screenshot from 2024-04-03 09-16-39

**Comment by Hermione2408** [comment]
Yes looks promising!

**Comment by nehagup** [comment]
@Aditya-eddy please ensure that the language logos are centre aligned (vert and horse) and are within the same height.

**Comment by Aditya-eddy** [comment]
Hey @nehagup I have commited the changes please check I think they look aligned in one row now!

File: components/language.tsx [hunk 2]
```diff
@@ -638,8 +638,25 @@ export default function Language() {
                 />
               </svg>
             </div>
+            <div className="flex items-center justify-center col-span-2 py-2 transition-transform hover:scale-110 md:col-auto">
+              <Image
+                src="/images/frameworks/csharp.svg" // Assuming the image file extension is .jpg
```
Comment (Hermione2408) [inline components/language.tsx]: the comment is irrelevant please remove them rest changes look fine!

**Review by Hermione2408 — CHANGES_REQUESTED** [review]

**Comment by Aditya-eddy** [comment]
Hey @Hermione2408 removed the comments!

**Review by Hermione2408 — APPROVED** [review]
LGTM @nehagup

**Review by nehagup — APPROVED** [review]

**Pull request merged**